setup.cfg

```diff
@@ -51,6 +51,15 @@ ignore_missing_imports = true
 [mypy-gunicorn.app.base]
 ignore_missing_imports = true
 
+[mypy-gunicorn.http.message]
+ignore_missing_imports = true
+
+[mypy-gunicorn.http.wsgi]
+ignore_missing_imports = true
+
+[mypy-gunicorn.workers.gthread]
+ignore_missing_imports = true
```

Comment:
Can you combine that with `[mypy-gunicorn.*]`?

Reply:
done